homeassistant/config_entries.py

```diff
@@ -2603,8 +2587,11 @@ async def async_forward_entry_setups(
             # If the lock was held when we stated, and it was released during
```

Comment:
```suggestion
            # If the lock was held when HA started, and it was released during
```
Maybe not add _we_, but talk it as a generic system?